ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -567,6 +578,7 @@ private[state] object Conversions {
           buildStatus(
             Code.ABORTED,
             s"Inconsistent: Missing input state for key ${rejection.getKey.toString}",
+            Map("key" -> rejection.getKey.toString),
```

Comment:
What does `toString` do here?

Reply:
As discussed offline, it uses `TextFormat.printer` underneath.